plugin/forward/forward.go

```diff
@@ -37,6 +38,9 @@ type Forward struct {
 	maxfails      uint32
 	expire        time.Duration
 
+	maxQueryCount int64
+	queryCount int64
```

Comment:
for sync atomic these need to be put first in the struct otherwise they don't align on ARM

Reply:
Thanks, fixed.